test/browser/run-test.sh

```diff
@@ -48,8 +48,8 @@ if [ "$TEST_OS" = "centos-8-stream" ]; then
     EXCLUDES="$EXCLUDES TestMachinesConsoles.testExternalConsole"
 fi
 
-# FIXME: https://github.com/cockpit-project/cockpit-machines/issues/581
-EXCLUDES="$EXCLUDES TestMachinesCreate.testCreateThenInstall"
+# depends on available host devices, TF machines not predictable enough
+EXCLUDES="$EXCLUDES TestMachinesHostDevs.testHostDevAdd"
```

Comment:
That may be necessary, but it should then go into its own commit (or possibly PR) with separate rationale. Did you see this fail recently?

Reply:
Fixed in #627.